websites/D/Discords.com/presence.ts

```diff
@@ -3,351 +3,516 @@ const presence = new Presence({
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
+enum Assets {
+	Logo = "https://i.imgur.com/ZjtyAQx.png",
+	Bot = "https://i.imgur.com/Z6ruEIy.png",
+	Heart = "https://i.imgur.com/bY8jPXF.png",
+}
+
 presence.on("UpdateData", async () => {
-	const showTimestamp = await presence.getSetting<boolean>("timestamp"),
-		showButtons = await presence.getSetting<boolean>("buttons"),
-		presenceData: PresenceData = {
-			largeImageKey: "discords_logo",
-		};
+	let presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+	};
 
-	if (document.location.pathname === "/")
+	const { href, pathname } = document.location,
+		[showTimestamp, showButtons] = await Promise.all([
+			presence.getSetting<boolean>("timestamp"),
+			presence.getSetting<boolean>("buttons"),
+		]),
+		pages: Record<string, PresenceData> = {
+			"/u/dashboard": {
+				details: "⚙️ Viewing: Dashboard",
```

Comment:
Let's not use emoji's.

Reply:
Awww ruining all the fun 😭

Reply:
> Awww ruining all the fun 😭

I think so @EncryptedDev is the admin here.

Reply:
Yeah, not too fond of the emojis and string. Agreed with DarkVillager and theusaf here